spaces/models.py

```diff
@@ -36,6 +37,44 @@ def __str__(self):
         return "{} ({})".format(self.name, self.parent.name if self.parent else "")
 
 
+class Unit(models.Model):
+    """
+    Model representation of Unit as in "office" or "premises" that could contain
+    separate building etc.
+    """
+
+    service_map_id = models.CharField(
+        verbose_name=_("Service map id"),
+        max_length=255,
+        unique=True,
+        blank=True,
+        null=True,  # If some units needs to be created manually.
+    )
+    name = models.CharField(verbose_name=_("Name"), max_length=255)
+    description = models.TextField(
+        verbose_name=_("Description"), max_length=255, blank=True, default=""
+    )
+    short_description = models.CharField(
+        verbose_name=_("Short description"), max_length=255, blank=True, default=""
+    )
+    web_page = models.URLField(
+        verbose_name=_("Homepage for the unit"), max_length=255, default=""
+    )
+    email = models.EmailField(verbose_name=_("Email"), max_length=255)
+    phone = models.CharField(verbose_name=_("Telephone"), max_length=255)
+    address = models.ForeignKey(
+        "applications.Address",  # Avoid circulars. Should address be moved to spaces?
```

Comment:
I think `Address` could somewhat general purpose class that could be used there and there when addresses are involved. 
What do you think?

Reply:
There is ´Location´ model in spaces app that includes address at the moment. It will also include coordinates (issue already in backlog). I think ´Unit´ should use the ´Location´  model as coordinates of Units could have some uses in the system (calculating distances etc).

Reply:
Location is a model that is designed to be used as OneToOne relation to various models which has location data.

Reply:
Excellent! :ok_hand:

Reply:
So I added coordinates to `Location` class and referenced `Unit` in there also.